### ISSUE TITLE
Subfamily Name/Stylename FAIL (Regular/Italic in VF)

### ISSUE THREAD

**Issue by RosaWagner** [googlefonts/fontbakery#3061]
## Observed behaviour

```
com.google.fonts/check/canonical_filename
 * FAIL: The file 'Texturina-Italic[opsz,wght].ttf' must be renamed to 'Texturina[opsz,wght].ttf' according to the Google Fonts naming policy for variable fonts. [code: bad-varfont-filename]
```

```
com.google.fonts/check/name/subfamilyname
 * FAIL: SUBFAMILY_NAME for Win "Regular" must be "Italic" [code: bad-familyname]
```

## Expected behaviour

So the first result is probably the consequence of the 2nd FAIL? I don't know how to resolve this. Although I am not sure if this second check is correct, otherwise we may have issues with other multi-axes variable having italics. I think this is linked to this ID25 issue where we'd better have "MyFontRoman-Regular" and "MyFontItalic-Italic". (ref: <https://github.com/huertatipografica/piazzolla/issues/12#issuecomment-681838915>)

## Resources and exact process needed to replicate

Things are happening in my fork for Texturina: https://github.com/RosaWagner/Texturina

**Comment by m4rc1e** [comment]
If I inspect the fonts, I get the following NameIDs

```
    <namerecord nameID="1" platformID="3" platEncID="1" langID="0x409">
      Texturina Medium Italic
    </namerecord>
    <namerecord nameID="2" platformID="3" platEncID="1" langID="0x409">
      Regular
    </namerecord>
```

What you actually want is:
```
    <namerecord nameID="1" platformID="3" platEncID="1" langID="0x409">
      Texturina Medium
    </namerecord>
    <namerecord nameID="2" platformID="3" platEncID="1" langID="0x409">
      Italic
    </namerecord>
```

> So the first result is probably the consequence of the 2nd FAIL?

Correct

> I don't know how to resolve this.

It's most likely a combination of having incorrect names in your designspace and .ufo sources. I'll see if I can find a better reference.

**Comment by felipesanches** [comment]
@m4rc1e please also consider suggesting better wording for these log result messages to make them clearer to the users.

**Comment by RosaWagner** [comment]
So it is probabmy a designspace file but we found a way to geerate mutli-axes variable fonts without a designspace files when sources are .glyphs… so no issue anymore. 

Although, for the ones using UFO sources and designspace files… the issue probably comes from the fact that the "default" master (or master origin) is not `Regular (400)`. I tried to correct the designspace file to see how to solve this (by adding correct `<stylemapfamilyname>` and `<stylemapstylename>` entries in the default source of the designspace), but it didn't work. So I don't know what better wording to suggest…
